File: solutions/java/src/Box.java
public class Box {
    public Box(int _width, int _height){
        width = _width;
        height = _height;
    }

    public Box(int _size){
        width = _size;
        height = _size;
    }

    public int width;

    public int height;

    public Pile pile;
}

File: solutions/csharp/PileBox/Box.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PileBox
{
	public class Box
	{
		public Box(int size)
		{
			Width = size;
			Height = size;
		}

		public Box(int width, int height)
		{
			Width = width;
			Height = height;
		}

		public int Height { get; set; }

		public int Width { get; set; }

		public Pile Pile { get; set; }
	}
}
